CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- MongoDB - Enhance authentication mechanism using optional salsprep library
```

Comment:
```suggestion
- MongoDB - Enhance authentication mechanism using optional `salsprep` library.
```

1. There is a final dot in all changelog lines.
2. (optional) I added backquotes to `salsprep` since it's a tech library name, but it's up to you!
3. (optional) It could belong to `### Added` section IMO, still @arnaudbesnier challenged this: "Is it a feature?". It's up to you too :).